woocommerce/wc-admin
# Admin Notes: Add new Note suggesting Jetpack Backup to WooCommerce users

As part of the effort to cross-promote Jetpack Backup to WooCommerce users, this PR adds a new Admin Note under the following conditions:
- Jetpack is installed
- Site does not have a Jetpack Backup product or plan
- WooCommerce has been enabled for at least a week

### Screenshots
<img width="530" alt="Screen Shot 2021-03-29 at 3 36 43 pm" src="https://user-images.githubusercontent.com/19717814/113435531-d7838e80-9397-11eb-9ad6-0e84effcaa7a.png">

### Detailed test instructions:

-   Create a test site that has Jetpack and any of the Jetpack plans/products which include Jetpack Backup (e.g. Jetpack Backup Daily, Jetpack Security Realtime, etc). You can use JN.
-   Either use a test site that is more than a week old or use the following SQL to make it so (you can use [this plugin](https://wordpress.org/plugins/wp-phpmyadmin-extension/)):
```
UPDATE `wp_options` SET `option_value`=UNIX_TIMESTAMP(DATE_SUB(NOW(), INTERVAL 10 day)) WHERE `option_name` = 'woocommerce_admin_install_timestamp';
```
-   Check out this branch and activate WooCommerce Admin.
-   Run the `wc_admin_daily` cron job. You can use a plugin like [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/).
-   Check the WooCommerce Inbox and ensure that the Note appears as in the screenshot.
-   Ensure the CTA button works as expected and takes you to the [custom landing page with the correct UTM params](https://jetpack.com/upgrade/backup-woocommerce/?utm_source=inbox&utm_medium=automattic_referred&utm_campaign=jp_backup_to_woo)
-  Remove the Jetpack Plan or change it to one that does NOT include Backup.
-  Run the cron job again and ensure that the Note no longer appears.
